{"uid":"ebbe89c50feffefb","name":"test_open_carcenter","fullName":"scripts.test01_carcenter.TestCarcenter#test_open_carcenter","historyId":"6e732a96989136ed4f9473cec8c0b131","time":{"start":1700635068688,"stop":1700635071824,"duration":3136},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":30,"retriesStatusChange":true,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestCarcenter","time":{"start":1700635057090,"stop":1700635068687,"duration":11597},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0}],"testStage":{"status":"passed","steps":[{"name":"设置页 - 点击设置","time":{"start":1700635068688,"stop":1700635070658,"duration":1970},"status":"passed","steps":[],"attachments":[{"uid":"409ba7d7298ec325","name":"点击设置","source":"409ba7d7298ec325.png","type":"image/png","size":1030958}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":1,"hasContent":true,"stepsCount":0},{"name":"关于 - 获取设置页面状态","time":{"start":1700635070658,"stop":1700635071824,"duration":1166},"status":"passed","steps":[],"attachments":[{"uid":"254ce917dc47008f","name":"查看设置页","source":"254ce917dc47008f.png","type":"image/png","size":613851}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":1,"hasContent":true,"stepsCount":0}],"attachments":[{"uid":"94846e58377bb9c","name":"log","source":"94846e58377bb9c.txt","type":"text/plain","size":143}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"hasContent":true,"stepsCount":2},"afterStages":[{"name":"_xunit_setup_method_fixture_TestCarcenter::0","time":{"start":1700635071824,"stop":1700635072115,"duration":291},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0}],"labels":[{"name":"severity","value":"blocker"},{"name":"parentSuite","value":"scripts"},{"name":"suite","value":"test01_carcenter"},{"name":"subSuite","value":"TestCarcenter"},{"name":"host","value":"H-K4E-234022"},{"name":"thread","value":"5580-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"scripts.test01_carcenter"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[{"uid":"6bfed459bdd6815b","status":"passed","time":{"start":1700634041369,"stop":1700634044237,"duration":2868}},{"uid":"a6b66fc89bb310c5","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: A new session could not be created. Details: java.lang.IllegalStateException: UiAutomation not connected, UiAutomation@56e647c[id=-1, flags=0]\nStacktrace:\nio.appium.uiautomator2.common.exceptions.SessionNotCreatedException: java.lang.IllegalStateException: UiAutomation not connected, UiAutomation@56e647c[id=-1, flags=0]\n\tat io.appium.uiautomator2.handler.NewSession.safeHandle(NewSession.java:60)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:77)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)\n\tat io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)\n\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)\n\tat io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)\n\tat io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)\n\tat io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)\n\tat java.lang.Thread.run(Thread.java:923)\nCaused by: java.lang.IllegalStateException: UiAutomation not connected, UiAutomation@56e647c[id=-1, flags=0]\n\tat android.app.UiAutomation.throwIfNotConnectedLocked(UiAutomation.java:1239)\n\tat android.app.UiAutomation.getServiceInfo(UiAutomation.java:497)\n\tat androidx.test.uiautomator.UiDevice.<init>(UiDevice.java:115)\n\tat androidx.test.uiautomator.UiDevice.getInstance(UiDevice.java:261)\n\tat io.appium.uiautomator2.utils.Device.getUiDevice(Device.java:34)\n\tat io.appium.uiautomator2.core.UiAutomatorBridge.getUiAutomation(UiAutomatorBridge.java:53)\n\tat io.appium.uiautomator2.core.UiAutomation.<init>(UiAutomation.java:32)\n\tat io.appium.uiautomator2.core.UiAutomation.getInstance(UiAutomation.java:37)\n\tat io.appium.uiautomator2.model.NotificationListener.<init>(NotificationListener.java:43)\n\tat io.appium.uiautomator2.model.NotificationListener.getInstance(NotificationListener.java:48)\n\tat io.appium.uiautomator2.model.settings.EnableNotificationListener.getValue(EnableNotificationListener.java:31)\n\tat io.appium.uiautomator2.model.settings.EnableNotificationListener.getValue(EnableNotificationListener.java:21)\n\tat io.appium.uiautomator2.model.settings.AbstractSetting.reset(AbstractSetting.java:57)\n\tat io.appium.uiautomator2.model.settings.Settings.resetForNewSession(Settings.java:73)\n\tat io.appium.uiautomator2.handler.NewSession.safeHandle(NewSession.java:52)\n\t... 34 more","time":{"start":1699928695942,"stop":1699928695942,"duration":0}},{"uid":"89fc310a9b528626","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: A new session could not be created. Details: java.lang.IllegalStateException: UiAutomation not connected, UiAutomation@56e647c[id=-1, flags=0]\nStacktrace:\nio.appium.uiautomator2.common.exceptions.SessionNotCreatedException: java.lang.IllegalStateException: UiAutomation not connected, UiAutomation@56e647c[id=-1, flags=0]\n\tat io.appium.uiautomator2.handler.NewSession.safeHandle(NewSession.java:60)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:77)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)\n\tat io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)\n\tat io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)\n\tat io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)\n\tat io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)\n\tat io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)\n\tat io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)\n\tat java.lang.Thread.run(Thread.java:923)\nCaused by: java.lang.IllegalStateException: UiAutomation not connected, UiAutomation@56e647c[id=-1, flags=0]\n\tat android.app.UiAutomation.throwIfNotConnectedLocked(UiAutomation.java:1239)\n\tat android.app.UiAutomation.getServiceInfo(UiAutomation.java:497)\n\tat androidx.test.uiautomator.UiDevice.<init>(UiDevice.java:115)\n\tat androidx.test.uiautomator.UiDevice.getInstance(UiDevice.java:261)\n\tat io.appium.uiautomator2.utils.Device.getUiDevice(Device.java:34)\n\tat io.appium.uiautomator2.core.UiAutomatorBridge.getUiAutomation(UiAutomatorBridge.java:53)\n\tat io.appium.uiautomator2.core.UiAutomation.<init>(UiAutomation.java:32)\n\tat io.appium.uiautomator2.core.UiAutomation.getInstance(UiAutomation.java:37)\n\tat io.appium.uiautomator2.model.NotificationListener.<init>(NotificationListener.java:43)\n\tat io.appium.uiautomator2.model.NotificationListener.getInstance(NotificationListener.java:48)\n\tat io.appium.uiautomator2.model.settings.EnableNotificationListener.getValue(EnableNotificationListener.java:31)\n\tat io.appium.uiautomator2.model.settings.EnableNotificationListener.getValue(EnableNotificationListener.java:21)\n\tat io.appium.uiautomator2.model.settings.AbstractSetting.reset(AbstractSetting.java:57)\n\tat io.appium.uiautomator2.model.settings.Settings.resetForNewSession(Settings.java:73)\n\tat io.appium.uiautomator2.handler.NewSession.safeHandle(NewSession.java:52)\n\t... 34 more","time":{"start":1699928690022,"stop":1699928690022,"duration":0}},{"uid":"838dd1befee2d413","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\n    at UIA2Proxy.command (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\jsonwp-proxy\\proxy.js:355:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at UiAutomator2Server.startSession (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\uiautomator2.js:260:5)\n    at AndroidUiautomator2Driver.startUiAutomator2Session (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\driver.js:430:5)\n    at AndroidUiautomator2Driver.createSession (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\driver.js:241:7)\n    at AppiumDriver.createSession (D:\\software\\nodejs\\node_global\\node_modules\\appium\\lib\\appium.js:352:35)","time":{"start":1699928679238,"stop":1699928679238,"duration":0}},{"uid":"284004924cb29ba3","status":"passed","time":{"start":1699856686413,"stop":1699856689595,"duration":3182}},{"uid":"edf20c42ba77ed05","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\n    at UIA2Proxy.command (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\jsonwp-proxy\\proxy.js:355:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at AndroidUiautomator2Driver.commands.getDevicePixelRatio (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\commands\\viewport.js:14:10)\n    at AndroidUiautomator2Driver.fillDeviceDetails (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\driver.js:256:28)\n    at AndroidUiautomator2Driver.createSession (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\driver.js:242:7)\n    at AppiumDriver.createSession (D:\\software\\nodejs\\node_global\\node_modules\\appium\\lib\\appium.js:352:35)","time":{"start":1699841947890,"stop":1699841947890,"duration":0}},{"uid":"cf9b79667ee90c9d","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\n    at UIA2Proxy.command (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\jsonwp-proxy\\proxy.js:355:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at AndroidUiautomator2Driver.commands.getDevicePixelRatio (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\commands\\viewport.js:14:10)\n    at AndroidUiautomator2Driver.fillDeviceDetails (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\driver.js:256:28)\n    at AndroidUiautomator2Driver.createSession (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\driver.js:242:7)\n    at AppiumDriver.createSession (D:\\software\\nodejs\\node_global\\node_modules\\appium\\lib\\appium.js:352:35)","time":{"start":1699841941362,"stop":1699841941362,"duration":0}},{"uid":"f213eb46dd0f0284","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details\n    at UIA2Proxy.command (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\jsonwp-proxy\\proxy.js:355:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at AndroidUiautomator2Driver.helpers.doFindElementOrEls (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\lib\\commands\\find.js:30:10)\n    at doFind (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\node_modules\\appium-android-driver\\lib\\commands\\find.js:36:17)\n    at wrappedCondFn (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\timeout.ts:137:14)\n    at spin (E:\\DM\\ShirleyPytest\\node_modules\\asyncbox\\lib\\asyncbox.js:221:20)\n    at waitForCondition (E:\\DM\\ShirleyPytest\\node_modules\\asyncbox\\lib\\asyncbox.js:238:10)\n    at AndroidUiautomator2Driver.implicitWaitForCondition (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\timeout.ts:139:12)\n    at AndroidUiautomator2Driver.findElOrEls (E:\\DM\\ShirleyPytest\\node_modules\\appium-uiautomator2-driver\\node_modules\\appium-android-driver\\lib\\commands\\find.js:60:5)\n    at AndroidUiautomator2Driver.findElOrElsWithProcessing (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\find.ts:60:12)\n    at AndroidUiautomator2Driver.findElement (E:\\DM\\ShirleyPytest\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\find.ts:75:12)","time":{"start":1699841940583,"stop":1699841940833,"duration":250}},{"uid":"8bd290fe04c8d922","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000017794AFAAD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1699841850632,"stop":1699841850632,"duration":0}},{"uid":"25dd0644854fbc98","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000017794BECD00>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1699841842295,"stop":1699841842295,"duration":0}},{"uid":"2daced2318713c8f","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000017793E33370>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1699841832761,"stop":1699841832761,"duration":0}},{"uid":"252e474b21a2b12","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000245947C6410>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1699841689929,"stop":1699841689930,"duration":1}},{"uid":"d70c644497e1b1e8","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024594897DC0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1699841681502,"stop":1699841681502,"duration":0}},{"uid":"8b365e3fa9a71e57","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024593BA7040>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1699841671917,"stop":1699841671917,"duration":0}},{"uid":"c8f7f0b191115541","status":"passed","time":{"start":1695199838589,"stop":1695199841575,"duration":2986}},{"uid":"13f9984e02d53b6f","status":"broken","statusDetails":"_pytest.fixtures.FixtureLookupError: ('args', <FixtureRequest for <Function test_open_carcenter>>)","time":{"start":1695199109657,"stop":1695199109657,"duration":0}},{"uid":"e8ea5707f347d6e2","status":"broken","statusDetails":"_pytest.fixtures.FixtureLookupError: ('args', <FixtureRequest for <Function test_open_carcenter>>)","time":{"start":1695199099512,"stop":1695199099512,"duration":0}},{"uid":"7552087c543803d4","status":"broken","statusDetails":"_pytest.fixtures.FixtureLookupError: ('args', <FixtureRequest for <Function test_open_carcenter>>)","time":{"start":1695199087443,"stop":1695199087443,"duration":0}},{"uid":"8e4fa4f0d22a2a24","status":"passed","time":{"start":1695197518517,"stop":1695197521376,"duration":2859}},{"uid":"eff759e64097fcda","status":"passed","time":{"start":1693299985343,"stop":1693299987022,"duration":1679}},{"uid":"b8f54fec3692bfb9","status":"passed","time":{"start":1693299819473,"stop":1693299821973,"duration":2500}},{"uid":"8d4f9b284671334a","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000273B4D3AB30>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1693299726705,"stop":1693299726705,"duration":0}},{"uid":"f513eba0f6b6956d","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000273B4E0CE80>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1693299717928,"stop":1693299717928,"duration":0}},{"uid":"35b9de351f5fe032","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000273B49AF1F0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1693299708186,"stop":1693299708186,"duration":0}},{"uid":"6be9c351064de10","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1693291250914,"stop":1693291250914,"duration":0}},{"uid":"be7d4e9b1b2b163a","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1693291248727,"stop":1693291248727,"duration":0}},{"uid":"a29e916222011513","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1693291246439,"stop":1693291246439,"duration":0}},{"uid":"1fd0b11cf3222b22","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1693291128482,"stop":1693291128482,"duration":0}},{"uid":"277dfa5ba04b934d","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1693291126347,"stop":1693291126347,"duration":0}},{"uid":"915ea3f9acca756b","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1693291123241,"stop":1693291123241,"duration":0}}],"categories":[],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":8,"unknown":0,"total":10},"items":[{"uid":"b1299f01e71e7458","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/16/allure/#testresult/b1299f01e71e7458","status":"passed","time":{"start":1695199838589,"stop":1695199841575,"duration":2986}},{"uid":"4717d2a1c6cac9a1","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/15/allure/#testresult/4717d2a1c6cac9a1","status":"passed","time":{"start":1695199838589,"stop":1695199841575,"duration":2986}},{"uid":"5f63e708457234db","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/14/allure/#testresult/5f63e708457234db","status":"passed","time":{"start":1695199838589,"stop":1695199841575,"duration":2986}},{"uid":"92f1653ce3ecc1dc","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/13/allure/#testresult/92f1653ce3ecc1dc","status":"passed","time":{"start":1695199838589,"stop":1695199841575,"duration":2986}},{"uid":"6d2500dfdbe8a0bf","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/12/allure/#testresult/6d2500dfdbe8a0bf","status":"broken","statusDetails":"_pytest.fixtures.FixtureLookupError: ('args', <FixtureRequest for <Function test_open_carcenter>>)","time":{"start":1695199109657,"stop":1695199109657,"duration":0}},{"uid":"154f423f59590122","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/11/allure/#testresult/154f423f59590122","status":"broken","statusDetails":"_pytest.fixtures.FixtureLookupError: ('args', <FixtureRequest for <Function test_open_carcenter>>)","time":{"start":1695199109657,"stop":1695199109657,"duration":0}},{"uid":"32f62c864f11ce45","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/10/allure/#testresult/32f62c864f11ce45","status":"passed","time":{"start":1695197518517,"stop":1695197521376,"duration":2859}},{"uid":"cb890e2e287353de","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/9/allure/#testresult/cb890e2e287353de","status":"passed","time":{"start":1693299985343,"stop":1693299987022,"duration":1679}},{"uid":"f9e57bc6c29ab2e3","reportUrl":"http://10.223.60.25:8080/job/LocalWorkspace/job/LocalShirleyPytest/8/allure/#testresult/f9e57bc6c29ab2e3","status":"passed","time":{"start":1693299985343,"stop":1693299987022,"duration":1679}}]},"tags":[]},"source":"ebbe89c50feffefb.json","parameterValues":[]}